src/apicache.js

```diff
@@ -102,28 +103,40 @@ function ApiCache() {
 
   function accumulateContent(res, content) {
     if (content) {
-      if (typeof(content) == "string") {
-        res._apicache.content=(res._apicache.content || "") + content;
-      }
-      else {
-        res._apicache.cacheable=false;
+      if (typeof(content) == 'string') {
+        res._apicache.content = (res._apicache.content || '') + content;
+      } else {
+        res._apicache.content = content
```

Comment:
Is there any concept of appending one buffer to another, or appending a buffer to a string for that matter?  

res.write("string")
res.write(buffer)
res.write(buffer2)
res.write("string2")

Is that a use case that needs to be considered?